### ISSUE TITLE
Upcoming breaking change in HA 2023.3

### ISSUE THREAD

**Issue by berndulum** [gody01/ecovent_v2#9]
It looks like there is an upcoming change in Home Assistant which requires small adjustments in this integration. I am using Home Assistant 2023.2.3 and get the following message when starting it with the ecovent_v2 integration enabled:

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77
First occurred: February 17, 2023 at 22:03:12 (1 occurrences)
Last logged: February 17, 2023 at 22:03:12

Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for ecovent_v2 using this method at custom_components/ecovent_v2/__init__.py, line 49: hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

Looks like in the future `async_forward_entry_setups` must be called instead of `async_setup_platforms`.

**Comment by gody01** [comment]
Thank You for reporting.
I will look into it.

**Comment by gody01** [comment]
It should be fixed in 0.9.2

**Comment by berndulum** [comment]
Thanks for the quick fix. The message is gone now.
